rules/windows/defense_evasion_code_injection_conhost.toml

```
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/08/31"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/31"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious Conhost child process which may indicate a code injection activity. Verify process details such as
+commandline, parent process and network connections.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious Conhost Child Process"
```

I first interpreted this the other way. I thought you meant that you were looking for an unusual child process named conhost. But since you're looking for a child process spawned _from_ conhost, I think this name might help.
```suggestion
name = "Suspicious Process from Conhost"
```